### ISSUE TITLE
Unable to upgrade from 5.4.1 to 5.5.0

### ISSUE THREAD

**Issue by jameswill** [revive-adserver/revive-adserver#1445]
I am getting following error on upgrading Revive 5.4.1 to 5.5.0 However the upgrade gets completed. Is this a bug ?

**Error: newer_exist : [[Path]]/www/admin/plugins/apVideoUI/navigation/videoBannerChecker.php
#! apVideo(): Existing file's timestamp is 14/09/2023 09:05:54 ... Replacement file's timestamp is 09/08/2023 09:31:40**

![image](https://github.com/revive-adserver/revive-adserver/assets/6368036/d2a0a8cf-e8b4-4cd2-a948-24f29111bed7)

I am using PHP 7.2 and Mysql 5.5

I have perform the upgrade with following steps:

1.  Safeguarded /var/default.conf.php and /var/domain.conf.php
2.  Safe guarded /plugins directory.
3. Extracting new package on the old one.
4. Visited /www/admin
5. On the last step of the upgrade it shows plugin **"apVideo" --> Failed to install** and throws the error.
6. On clicking error_list and /var/install.log files it shows the error mentioned. (Screenshot attached)

**Comment by mbeccati** [comment]
> Extracting new package on the old one.

Please follow the upgrade guide. That's not how you should upgrade.

**Comment by erikgeurts** [comment]
The full instructions on how to perform an update can be found at https://www.revive-adserver.com/how-to/update/

**Comment by jameswill** [comment]
Hi,

Thank you for the guidance and link.

I have tried upgrading my existing installation using the guide and have few questions which I did not find in the upgrade docs.

1. If I want to upgrade the installation with same DB Is that possible ? As "Creating a new database (optional)" step is optional.
  If that is possible and I simply copy the old files /var/default.conf.php and /var/domain.conf.php to the new location the upgrader does not ask for "configuration path" for the old version and gives same error "apVideo" --> Failed to install I mentioned above.


> The Revive Adserver configuration file can be found in the ‘var’ folder just below the ‘adserver’ folder. The name of the starts with your domain name and ends in ‘conf.php’. So for our fictional installation at www.example.com/adserver, the configuration file is named www.example.com.conf.php.

Also the guide you provided only say to create the "www.example.com.conf.php" file. What about **default.conf.php** which contains realConfig name ?

I also tried upgrading creating a new DB and inserting it's details in www.example.com.conf.php file and it throws following error while upgrade.

![image](https://github.com/revive-adserver/revive-adserver/assets/6368036/b5373234-f2ac-4d5b-b492-8a1fedc9b5db)


Let me know I am doing anything wrong.
